Flepper.QueryBuilder.DapperExtensions/Flepper.QueryBuilder.DapperExtensions.csproj

```diff
@@ -19,6 +19,10 @@
     <DebugSymbols>true</DebugSymbols>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
C# 7.1 only in debug?

Reply:
https://github.com/Flepper/flepper/pull/90/commits/ce217664edfba78c24b7b1351858097a8cb405b5